Cue/app/tabs/library/LibraryHome.ios.js

```diff
@@ -82,10 +82,8 @@ class LibraryHome extends React.Component {
              this.props.navigator.push({deck})
            }},
           ],
-          {
-            type: 'plain-text',
-            placeholder: 'My Great Deck',
-          },
+          type: 'plain-text',
```

Comment:
This still doesn't work, `'plain-text'` and `'My Great Deck'` have to be just normal parameters.